index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
```

Comment:
This is not needed here, you can just write `@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto';`in the application.scss before `styles` import